Swat/SwatString.php

```diff
@@ -2379,20 +2268,21 @@ private static function parseNegativeNotation($string)
                 }
                 break;
 
-            // negative sign trails number: 5.00-
+                // negative sign trails number: 5.00-
```

Comment:
this indentation is wrong

Reply:
I think because it assumes the comment belongs to the case block it's in, and not the one that starts on the next line.

I can "fix" this so `php-cs-fixer` doesn't complain with either adding the comment to the same line as the `case`:
```php
case 2: // negative sign trails number: 5.00-
    ...
```
or moving it inside the block:
```php
case 2:
    // negative sign trails number: 5.00-
    ...
```